src/components/course/routes/CoursePageRoutes.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Switch, Route, useRouteMatch } from 'react-router-dom';
+
+import CourseAbout from './CourseAbout';
+import ExternalCourseEnrollment from './ExternalCourseEnrollment';
+import ExternalCourseEnrollmentConfirmation from './ExternalCourseEnrollmentConfirmation';
+
+const CoursePageRoutes = () => {
+  const { path: baseCoursePageRoutePath } = useRouteMatch();
+  return (
+    <Switch>
+      <Route exact path={baseCoursePageRoutePath} component={CourseAbout} />
+      <Route exact path={`${baseCoursePageRoutePath}/enroll`} component={ExternalCourseEnrollment} />
+      <Route exact path={`${baseCoursePageRoutePath}/enroll/complete`} component={ExternalCourseEnrollmentConfirmation} />
```

Comment:
@adamstankiewicz TODO: swap this for `PageRoute` components to ensure we get page views events in Segment when navigating between them.